amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -260,6 +260,12 @@ public class AmoroManagementConf {
           .defaultValue("jdbc:derby:/tmp/amoro/derby;create=true")
           .withDescription("Database connection address");
 
+  public static final ConfigOption<Boolean> DB_INIT_SCHEMA =
+      ConfigOptions.key("database.init-schema")
```

Comment:
Is `database.auto-create-tables` a proper name?

Reply:
BTW, we may add this new configuration to our example configuration file.

Reply:
fixed